libcni/api.go

```diff
@@ -817,6 +817,8 @@ func (c *CNIConfig) GCNetworkList(ctx context.Context, list *NetworkConfigList,
 		}
 		if args != nil {
 			inject["cni.dev/valid-attachments"] = args.ValidAttachments
+			// GH-1101: spec used incorrect variable name
+			inject["cni.dev/attachments"] = args.ValidAttachments
```

Comment:
Looks good to me from cording perspective, but how about to change `#1101` from JIRA-ish name?

Reply:
Fixed.